catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -109,6 +113,17 @@ public void init() {
           res.header("Content-Encoding", "gzip");
         });
 
+    get(
+        "/cql/transforms",
+        (req, res) -> {
+          List transformers =
+              queryResponseTransformers
+                  .stream()
+                  .map(serviceReference -> serviceReference.getProperty(TRANSFORMER_ID_PROPERTY))
+                  .collect(Collectors.toList());
+          return mapper.toJson(transformers);
```

Comment:
Should this endpoint return all of the information about each transformer (title, description, mimetype(s), shortname, etc.)?

This could also be put into a separate endpoint like `/cql/transform/{transformer_id}`

Reply:
that shouldn't be needed for the scope of this ticket.

just the info needed for the ui dropdown should be returned, in this case the title is fine. if we need more complexity in the ui in the future we can return more details about the transformers.